src/test/java/uk/gov/justice/hmpps/prison/repository/AgencyRepositoryTest.java

```diff
@@ -201,6 +201,12 @@ public void testFilterPrisonIepReviewByIepLevel() {
         assertThat(results.getItems()).containsExactly(OFFENDER_6_IEP_REVIEW);
     }
 
+    @Test
+    public void testGetAgencyLocationsWhenAllOffendersSuspended() {
+        final var locations = repository.getAgencyLocationsBooked("BXI", LocalDate.of(2021, Month.MAY, 1), null);
+        assertThat(locations).hasSize(2);
```

Comment:
Might be worth asserting that you are getting the records back that you think you are?  Probably ok now but if more test data goes in it might start passing by accident?

Reply:
Thanks @mikehalmamoj - good point,  will make the change